src/axom/spin/ImplicitGrid.hpp

```diff
@@ -50,7 +54,7 @@ namespace spin
  * is designed for quick indexing and searching over a static (and relatively
  * small index space) in a relatively coarse grid.
  */
-template <int NDIMS, typename TheIndexType = int>
+template <int NDIMS, typename ExecSpace = axom::SEQ_EXEC, typename TheIndexType = int>
```

Comment:
This change might impact users. 
Please note in the `RELEASE-NOTES` that the template parameters for `ImplicitGrid` changed.